sound/hybrid_font.h

```diff
@@ -234,11 +234,38 @@ class HybridFont : public SaberBase, public Looper {
     EnableAmplifier();
     if (!f->files_found()) return RefPtr<BufferedWavPlayer>(nullptr);
     RefPtr<BufferedWavPlayer> player = GetFreeWavPlayer();
-    if (player) {
-      player->set_volume_now(font_config.volEff / 16.0f);
-      player->PlayOnce(f);
-      current_effect_length_ = player->length();
+    if (!player) {
+      STDOUT.println("Out of WAV players! Getting more...");
+      float oldest_ = 0;
+      uint32_t kill_ = 0;
```

Comment:
make this an "int", and then change 0 to -1 so that we can recognize when nothing was found below. I also think that we should also count how many players are playing  the same sound and only do the killing if the number of players is > 1, otherwise it's going to be too obvious.

Reply:
> make this an "int", and then change 0 to -1 so that we can recognize when nothing was found below.

I was thinking with the un-hardcoding of the i == 6 above, to just do ` if (i == NELEM(wav_players))` 
In testing with printouts, I saw the for loop cycle 0-6 over and over when not getting a sound, therefore never updating kill (now player_to_restart). So checking if we've changed that int wouldn't work. Well, not the way it's written currently anyway.

> I also think that we should also count how many players are playing the same sound and only do the killing if the number of players is > 1, otherwise it's going to be too obvious.

It doesn't seem possible we'd ever kill off a single instance of a sound though, no?
With 7 wavplayers,
a max of 4 tied up with non-effects 
we're spamming new effect events, using the last 3 or 4 players,
the oldest player_to_restart should always be 3 or 4 players older than our current sound.

Reply:
ok